ERC-20/contracts/LeoToken.sol

```diff
@@ -20,7 +35,17 @@ contract LeoToken {
     }
 
     modifier hasEnoughAmount( address _sender, uint256 _value ) {
-        require( balanceOf[ _sender ] >= _value, 'Not enough tokens' );
+        // solhint-disable-next-line
+        if( vestingDate[ _sender ] > block.timestamp && balanceOf[ _sender ] >= _value ) {
```

Comment:
Solhint wywala mi błąd, ze względu na możliwe oszustwo z block.timestamp, wiem o co chodzi, ale ciekawości znasz jakieś zabezpieczenia / inne metody?

Reply:
Nie ma. A oszustwo nie jest możliwe, bo dla Ciebie najważniejsza jest chronologia. Gdyby tu ktoś oszukał, to reszta sieci i tak to odrzuci.